packages/block-editor/src/hooks/test/__snapshots__/align.native.js.snap

```diff
@@ -2,19 +2,19 @@
 
 exports[`Align options for group block sets Full width option 1`] = `
 "<!-- wp:group {"align":"full","layout":{"type":"constrained"}} -->
-<div class="wp-block-group alignfull"></div>
+<div class="wp-block-group-group wp-block-group alignfull"></div>
```

Comment:
It'd be great if we were able to keep the default block class name as the first classname, and add the variation as the second one (rather than vice versa). Is this happening due to the order that filters are run? Can we change that? 🤔

Reply:
Ah, I think I see it now. We're adding the filters in the correct order, but each of them is [prepending](https://github.com/WordPress/gutenberg/pull/61864/files#diff-5f22902b77331fe4326ed58f63bef95005aef70a958db2de34e2d39a763ade80R41-R49) the generated block and variation class name, respectively -- meaning that when the variation class name filter is called last, the variation class name is prepended to everything else, including the block class name.